tests/utils/net_test.py

```diff
@@ -33,3 +32,15 @@ def test_download_from_s3_bucket(self, download):
                         expected_size, progress_indicator)
         download.assert_called_once_with("mybucket.elasticsearch.org", "data/documents.json.bz2",
                                          "/tmp/documents.json.bz2", expected_size, progress_indicator)
+
+    def test_progress(self):
+        progress = net.Progress("test")
+        mock_progress = mock.Mock()
+        progress.p = mock_progress
+        progress(42, 100)
+        assert mock_progress.print.called
+        mock_progress.reset_mock()
+        progress(42, None)
+        assert mock_progress.print.called
+
+
```

Comment:
`make precommit` has failed with:

```

03:59:48 ************* Module tests.utils.net_test
03:59:48 tests/utils/net_test.py:46:0: C0305: Trailing newlines (trailing-newlines)
```

can you please have a look?

Reply:
https://github.com/elastic/rally/pull/885/commits/5a64bfe1e131336b8782dc55ac1e0b5baa145521